docs/sources/setup-grafana/configure-grafana/configure-custom-branding/index.md

```diff
@@ -136,4 +132,4 @@ To customize the footer of a public dashboard, add the following section to the
 ;header_logo_hide =
 ```
 
-If you specify `footer_hide` to `true`, all the other values are ignored because the footer will not be shown.
+`footer_hide` is no longer available. Please set either a `footer_text` or a `footer_logo`; else it will default to the Grafana footer.
```

Comment:
@imatwawana , can you please look over the verbage?

Reply:
```suggestion
`footer_hide` is no longer available. Set either a `footer_text` or `footer_logo`. Otherwise, the default Grafana footer is displayed.
```